minigunapp/views.py

```diff
@@ -0,0 +1,36 @@
+from django.shortcuts import render
+from django.http import HttpResponse, Http404, JsonResponse
+from django.db import models
+
+import json
+import yaml
+from cerberus import Validator
+from minigunapp.models import Email
+from minigunapp.tasks import send_email
+
+v = Validator()
+schema = yaml.load(open('minigunapp/schemas/email.yaml'))
+
+def email(request):
+    if request.method in ['GET', 'POST']:
+        return globals()[request.method.lower() + '_email'](request) # WTB a real router
+    raise Http404("Not found")
+
+def get_email(request):
+    emails = [json.loads(str(email)) for email in Email.objects.all()]
```

Comment:
Bad things. I'm going to paginate this route. Makes me wonder if the django orm has some built-in pagination cleverness. I'll have to see...